### DISCUSSION TITLE
version control for collection object updates

### DISCUSSION THREAD

**Discussion by normanmadrid** [pocketbase/pocketbase#2988] [Q&A]
Has this been suggested already? It would be cool if we have even a basic log for what changes a collection object has gone through since creation. Would this be hard to implement?

**Comment by ganigeorgiev** [comment]
We support [Migrations](https://pocketbase.io/docs/migrations/) (both Go and JS) that are intended to describe and keep track of your collections structure changes (_with the next release there will be more documentation about this_).

If you are asking for collection records log - there is no such functionality builtin as it will be hard to generalize since the implementation will depend on the granularity of the stored changes and how the developer will interact with them. You can find a very rudimentary example in https://github.com/pocketbase/pocketbase/discussions/1281.
